qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -1206,3 +1207,128 @@ def new_modulated_gaussian_control(
         detunings=np.zeros(segment_count),
         durations=np.array([segment_duration] * segment_count),
     )
+
+
+def new_drag_control(
```

Comment:
My instinct is to proceed according to existing implementations (which AFAIK is just Qiskit and maybe this: https://github.com/BBN-Q/QGL) when it comes to convention, but it's true that DRAG is a whole family of techniques. Currently I believe I have all the relevant information on the implementation in the Notes section, but it may also be appropriate to make some of those details more prominent.

Reply:
It is enough to make it clear in the docstring that this is first order DRAG (Gaussian + derivative). Commonly DRAG is often short for first order DRAG, e.g., the Qiskit documentation.